search-endpoint.md

```diff
@@ -0,0 +1,397 @@
+# Discovery Search Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+## Search Endpoint
+**/api/discover/search**   
```

Comment:
/api/discover/search is fine for me as this is a slightly different endpoint than any other implemented so far. It is not really a way to expose resources it is a service / action.